README.md

```diff
@@ -4,12 +4,12 @@ Publisher Ads Audits for Lighthouse is a tool to improve ad speed and overall qu
 
 Publisher Ads Audits for Lighthouse is a plugin for [Lighthouse](https://github.com/GoogleChrome/lighthouse), an open-sourced tool integrated into Chrome dev tools that is widely used by developers.
 
-We are currently in alpha and still in the development process. In order to help us improve please [file an issue](https://github.com/googleads/ad-speed-insights/issues) to let us know of any issues or suggestions you may have.
+We are currently in alpha and still in the development process. In order to help us improve please [file an issue](https://github.com/googleads/pub-ads-lighthouse-plugin/issues) to let us know of any issues or suggestions you may have.
 
 ## Setup
 
 ```sh
-git clone git@github.com:googleads/ad-speed-insights.git
+git clone git@github.com:googleads/pub-ads-lighthouse-plugin.git
 cd ad-speed-insights
```

Comment:
cd pub-ads-lighthouse-plugin. There may be other instances, have you tried searching for ad.*speed.*insights in the codebase?